pkg/virtctl/vm/restart.go

```diff
@@ -0,0 +1,80 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
```

Comment:
I think start, stop and restart should be at the same file.

Reply:
Same as above.